projects/plugins/jetpack/modules/masterbar/inline-help/class-inline-help.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Inline Help
+ *
+ * Handles providing a LiveChat icon within WPAdmin until such time
+ * as the full live chat experience can be run in a non-Calypso environment.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+use Automattic\Jetpack\Redirect;
+use Automattic\Jetpack\Status;
+
+/**
+ * Class Inline_Help.
+ */
+class Inline_Help {
+
+	/**
+	 * Inline_Help constructor.
+	 */
+	public function __construct() {
+		$this->register_actions();
+	}
+
+	/**
+	 * Registers actions
+	 *
+	 * @return void
+	 */
+	public function register_actions() {
+
+		// Do not inject the FAB icon on embedded screens since the parent window may already contain a FAB icon.
+		$is_framed = ! empty( $_GET['frame-nonce'] );
+
+		if ( $is_framed ) {
+			return;
+		}
+
+		add_action( 'admin_footer', array( $this, 'add_fab_icon' ) );
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'add_fab_styles' ) );
+	}
+
+	/**
+	 * Outputs "FAB" icon markup and SVG.
+	 *
+	 * @return void|string the HTML markup for the FAB or early exit.
+	 */
+	public function add_fab_icon() {
+
+		if ( wp_doing_ajax() ) {
+			return;
+		}
+
+		$svg_allowed = array(
+			'svg'   => array(
+				'id'              => true,
+				'class'           => true,
+				'aria-hidden'     => true,
+				'aria-labelledby' => true,
+				'role'            => true,
+				'xmlns'           => true,
+				'width'           => true,
+				'height'          => true,
+				'viewbox'         => true, // <= Must be lower case!
+			),
+			'g'     => array( 'fill' => true ),
+			'title' => array( 'title' => true ),
+			'path'  => array(
+				'd'    => true,
+				'fill' => true,
+			),
+		);
+
+		$gridicon_help = file_get_contents( __DIR__ . '/gridicon-help.svg', true );
+
+		echo '<div class="inline-help"><a href="https://wordpress.com/help" target="_blank" rel="noopener noreferrer" title="' . esc_attr__( 'Help', 'jetpack' ) . '" class="inline-help__button">' . wp_kses( $gridicon_help, $svg_allowed ) . '</a></div>';
```

Comment:
Could we use a more specific name to avoid potential conflicts with any other plugins that may style and / or add `inline-help` containers to wp-admin?

Reply:
I agree. This is wide open. Lets try `.a8c-faux-inline-help`?

Reply:
Sounds good!

Reply:
Done